clang/lib/Sema/SemaBounds.cpp

```diff
@@ -2211,14 +2226,21 @@ namespace {
                                     CheckingState &State) {
       Expr *LHS = E->getLHS();
       Expr *RHS = E->getRHS();
+      ExprEqualMapTy SubExprGs;
 
-      // Infer the lvalue or rvalue bounds of the LHS.
+      // Infer the lvalue or rvalue bounds of the LHS,
```

Comment:
For multi-line comments, please be sure to use most of the 80 character line (say 70 characters).   The clang/LLVM coding convention encourages this.